mappings/net/minecraft/entity/EntityManager.mapping

```diff
@@ -1,4 +1,5 @@
-CLASS net/minecraft/class_5569
+CLASS net/minecraft/class_5569 net/minecraft/entity/EntityManager
```

Comment:
Lol no, this is an entity listener/callback assigned by the manager to each entity.
See https://github.com/FabricMC/yarn/pull/1967/files#diff-ef57025da02812a1e3d26e02f5612f228e64800e7afc5a84ef00d55818833599R1